frontend/integration-tests/views/kubevirt/login.view.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-undef, no-unused-vars */
```

Comment:
Remove this comment

Reply:
Why is there a special kubevirt login view? We already have a login view

Reply:
It's there because of a bug in web-ui, the logout does not return to login page which cause `logout` in view/login.view.ts does not work, this login view removes the last line.

But it should not a problem once we're using the some console, will remove it once I get a test for it asap.